includes/class-dispatcher.php

```diff
@@ -104,8 +104,24 @@ private static function send_activity_to_followers( $activity, $actor_id, $outbo
 
 		$json = $activity->to_json();
 
+		// We will store the json as generated by the transformer, even though it 's also in plaintext in $outbox_item->post_content
+		// This will also allow us to keep the logs below leaner.
+		\add_post_meta( $outbox_item->ID, 'activitypub_sent_json', $json );
+		// This will allow error checking later, that all inboxes have been sent.
+		\add_post_meta( $outbox_item->ID, 'activitypub_sent_inboxes', $inboxes, true );
```

Comment:
shouldn't we simply keep a list of inboxes that bounce? this is a lot of data we store per Outbox Activity!?!

Reply:
I wanted a fully debuggable picture of what happens, given that the list of inboxes POSTed to for any given outbox activity will be somewhat dynamically generated. At the same time I was envisioning using it as a basis for rolling batching, which I did not get to. 

But since we're not doing batching yet either, and all of the inboxes are stored in the logs below, we don't need it right now.